watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/electrochemical_nutrient_removal/electrochemical_nutrient_removal_ui.py

```diff
@@ -330,6 +341,28 @@ def export_variables(flowsheet=None, exports=None):
     )
 
     # System metrics
+    exports.add(
+        obj=fs.costing.LCOT,
+        name="LCOT",
+        ui_units=fs.costing.base_currency / pyunits.m**3,
+        display_units="$/m3 of feed",
+        rounding=3,
+        description="Levelized cost of treatment including operating and capital costs",
+        is_input=False,
+        is_output=True,
+        output_category="Levelized cost metrics",
+    )
+    exports.add(
+        obj=fs.costing.LCOT_with_revenue,
+        name="LCOT with revenue",
+        ui_units=fs.costing.base_currency / pyunits.m**3,
+        display_units="$/m3 of feed",
+        rounding=3,
+        description="Levelized cost of treatment including revenue of struvite and disposal costs for MgCl2 and polymer",
```

Comment:
Does this model have any polymer? Also, should this be "chemical consumption costs for MgCl2" instead of "disposal costs for MgCl2"?

Reply:
There's no polymer in this example

Reply:
I think it is no need to memtion MgCl2 cost, it's already included in the LCOT no matter with or without revenue.

Reply:
I think it can't hurt to explicitly state the MgCl2 cost, but I can get rid of it if that's what we want.